ansible/roles/kubernetes/tasks/gen_certs.yml

```diff
@@ -30,7 +36,7 @@
     creates: "{{ kube_cert_dir }}/server.crt"
   environment:
     MASTER_IP: "{{ kube_cert_ip }}"
-    MASTER_NAME: "{{ inventory_hostname }}"
+    MASTERS: "{{ groups['masters']|join(",") }}"
```

Comment:
The error appears to have been in 'contrib/ansible/roles/kubernetes/tasks/gen_certs.yml': line 39, column 42, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    MASTER_IP: "{{ kube_cert_ip }}"
    MASTERS: "{{ groups['masters']|join(",") }}"
                                         ^ here
We could be wrong, but this one looks like it might be an issue with
missing quotes.  Always quote template expression brackets when they
start a value. For instance:

    with_items:
      - {{ foo }}

Should be written as:

    with_items:
      - "{{ foo }}"